includes/css/pattern-styles.css

```diff
@@ -860,6 +860,11 @@ h2 > a {
 }
 
 /* Spotlight Cards */
+.spotlight-float-image {
+  float: left;
+  margin: 0 1rem 1rem 0 !important;
```

Comment:
Only use !important to override inline styles. You can remove any inline styles on this element.